### DISCUSSION TITLE
New black hole and trail

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3164] [General]
This big black hole and it's trail would be nice to be in Stellarium. Does it have any classification or id?

https://hubblesite.org/contents/news-releases/2023/news-2023-010

**Comment by 10110111** [comment]
The paper doesn't appear to designate it anyhow, but it does provide the coordinates. Also, don't be fooled by the illustration in the article you linked to. The actual image is far less spectacular (see figure 1 in the paper).

**Comment by gzotti** [comment]
The nice picture in the PR shows an illustration, i.e., artwork. And sorry, we cannot add images from all press releases out there (or worse, just any random PRs somebody suggests). Stellarium's main task is in the unaided and small/medium telescope visual spectrum realm.
